src/cpu/aarch64/jit_uni_batch_normalization.cpp

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
 * Copyright 2020 Intel Corporation
```

Comment:
```suggestion
* Copyright 2020-2021 Intel Corporation
```

Reply:
Thank you for the suggestion. I'll fix it.